docs/querying.md

```diff
@@ -160,54 +165,139 @@ as well.
+[Op.noExtendRight]: [1, 2] // &< [1, 2) (PG range does not extend to the right of operator)
+[Op.noExtendLeft]: [1, 2]  // &> [1, 2) (PG range does not extend to the left of operator)
 ```
 
-### Combinations
+#### Combinations
 ```js
+const Op = Sequelize.Op;
+
 {
   rank: {
-    $or: {
-      $lt: 1000,
-      $eq: null
+    [Op.or]: {
+      [Op.lt]: 1000,
+      [Op.eq]: null
     }
   }
 }
 // rank < 1000 OR rank IS NULL
 
 {
   createdAt: {
-    $lt: new Date(),
-    $gt: new Date(new Date() - 24 * 60 * 60 * 1000)
+    [Op.lt]: new Date(),
+    [Op.gt]: new Date(new Date() - 24 * 60 * 60 * 1000)
   }
 }
 // createdAt < [timestamp] AND createdAt > [timestamp]
 
 {
-  $or: [
+  [Op.or]: [
     {
       title: {
-        $like: 'Boat%'
+        [Op.like]: 'Boat%'
       }
     },
     {
       description: {
-        $like: '%boat%'
+        [Op.like]: '%boat%'
       }
     }
   ]
 }
 // title LIKE 'Boat%' OR description LIKE '%boat%'
 ```
 
+#### Operators Aliases
+Sequelize allows setting specific strings as aliases for operators -
+```js
+const Op = Sequelize.Op;
+const operatorsAliases = {
+  $gt: Op.gt
+}
+const connection = new Sequelize(db, user, pass, { operatorsAliases })
+
+[Op.gt]: 6 // > 6
+$gt: 6 // same as using Op.gt (> 6)
+```
+
+
+#### Operators security
+Using Sequelize without any aliases improves security.
+Some frameworks automatically parse user input into js objects and if you fail to sanitize your input it might be possible to inject an Object with string operators to Sequelize.
+
+Not having any string aliases will make it extremely unlikely that operators could be injected but you should always properly validate and sanitize user input.
+
+For backward compatibility reasons Sequelize sets the following aliases by default -
+$eq, $ne, $gte, $gt, $lte, $lt, $not, $in, $notIn, $is, $like, $notLike, $iLike, $notILike, $regexp, $notRegexp, $iRegexp, $notIRegexp, $between, $notBetween, $overlap, $contains, $contained, $adjacent, $strictLeft, $strictRight, $noExtendRight, $noExtendLeft, $and, $or, $any, $all, $values, $col
+
+currently the following legacy aliases are also set but are planned to be fully removed in the near future -
+ne, not, in, notIn, gte, gt, lte, lt, like, ilike, $ilike, nlike, $notlike, notilike, .., between, !.., notbetween, nbetween, overlap, &&, @>, <@
+
+Due to the security improvements it is highly advised to only use the specific aliases you need.
+
+```js
+const Op = Sequelize.Op;
+
+//use sequelize without any operators aliases
+const connection = new Sequelize(db, user, pass, { operatorsAliases: {} });
+
+//use sequelize with only alias for $and => Op.and
+const connection2 = new Sequelize(db, user, pass, { operatorsAliases: { $and: Op.and } });
+```
+
+Sequelize will warn you if your using the default aliases and not limiting them
+if you want to keep using all default aliases (excluding legacy ones) without the warning you can pass the following operatorsAliases option -
```

Comment:
I am not ok with this, alias like `$gte` should always be available without user doing anything or passing anything.

String based alias on other hand should be available for bc, but can be turned off by flag

Reply:
The thing is '$gte' is still string based.
all aliases are available by default. but if you don't actually select the ones you need you'll see a warning. but everything will work as before.

There is an easier way to set all the '$' string aliases since Op actually exposes the setting. 
But I thought it would be a better practice for the user to set them and so know which ones he is actually enabling (If they really lazy they'll probably just copy past the aliases from this docs).
Other than kinda forcing the users to be aware of the aliases it also gives a bit easier eay to introduce future upgrades, if in a future version we want to completely remove or change the default aliases users who follow the code here will still get the same aliases they had has before with no breaking changes. So I think its a bit more future proof than relying on Sequelize setting it up for them